specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2020-11-01-preview/examples/Deployments_CreateOrUpdate.json

```diff
@@ -9,8 +9,10 @@
           "artifactSelector": "sub-module-1"
         },
         "deploymentSettings": {
-          "cpu": 1,
-          "memoryInGB": 3,
```

Comment:
Is the removing of these properties intentional? I still see them in the deploymentSettings response model.

Reply:
Yep, it is intentional. According to current design, the users can use either cpu/memory or resourceRequests to represent his App's size. It is a straight either or, they don't need to set them all.